docs/developers/command-line/input.md

```diff
@@ -0,0 +1,114 @@
+The Xbox uses so called [Xbox Input Devices (XID)](http://xboxdevwiki.net/Xbox_Input_Devices).
+
+In all cases, start by making sure you have the `-usb` option specified on the
+XQEMU command line.
+To connect multiple gamepads you can simply specify multiple `-device`.
+
+To find out more about QEMU USB emulation you can read [the QEMU User Documentation](http://qemu.weilnetz.de/qemu-doc.html#pcsys_005fusb).
+
+## Connecting a XID
+
+To connect a device to the virtual Xbox you must specify the driver for the
+emulated USB device and the port the device should connect to.
+
+The ports which can be used in XQEMU are:
+
+| Xbox Port | XQEMU USB-Port         |
+| :-------: | :--------------------- |
+| Player 1  | `bus=usb-bus.0,port=3` |
+| Player 2  | `bus=usb-bus.0,port=4` |
+| Player 3  | `bus=usb-bus.0,port=1` |
+| Player 4  | `bus=usb-bus.0,port=2` |
```

Comment:
Might need a future PR, but this table was reported on discord as incorrect recently.

Reply:
I don't think it's incorrect?
However, please create a separate issue about this. This moves content, but doesn't really modify it.